config/amqp.json

```diff
@@ -0,0 +1,32 @@
+{  
+   "request":{  
+      "method":"POST",
+      "path":"/amqp/*.json"
+   },
+   "response":{  
+      "statusCode":202,
+      "headers":{  
+         "Content-Type":[  
+            "application/json"
+         ]
+      },
+      "body":"{{persist.entity.content}}"
+   },
+   "persist": {
```

Comment:
I think in the configurations we should first add the "persist" node before the response, as the persistence is being processed first and the persist.entity.content is used in the response. However this is just for better reading of the configuration.